### PR TITLE
*: remove travis ci

### PR DESCRIPTION
**Pull request by mccxj**
for ##2451

travis-build dir is used by circle.yml, maybe we should rename it. 
Shall we?

### CODE REVIEW

**Comment by siddontang** [comment]
maybe we should rename it to ci-build dir. /cc @BusyJay

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## TiKV is a distributed Key-Value database powered by Rust and Raft
 
 
-[![Build Status](https://travis-ci.org/pingcap/tikv.svg?branch=master)](https://travis-ci.org/pingcap/tikv) [![CircleCI Status](https://circleci.com/gh/pingcap/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://circleci.com/gh/pingcap/tikv) [![Coverage Status](https://coveralls.io/repos/github/pingcap/tikv/badge.svg?branch=master)](https://coveralls.io/github/pingcap/tikv)
+[![CircleCI Status](https://circleci.com/gh/pingcap/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://circleci.com/gh/pingcap/tikv) [![Coverage Status](https://coveralls.io/repos/github/pingcap/tikv/badge.svg?branch=master)](https://coveralls.io/github/pingcap/tikv)
```
Comment (siddontang) [inline README.md]: `Build Status` may be better

**Comment by siddontang** [comment]
LGTM

**Comment by mccxj** [comment]
why jenkins failed?

**Comment by BusyJay** [comment]
It's not related to your changes.

**Pull request merged**